core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -389,7 +390,8 @@ var TopicEditorPage = function() {
       closeSaveModalButton,
       'Close save modal button takes too long to be clickable');
     await closeSaveModalButton.click();
-    await waitFor.pageToFullyLoad();
+    await waitFor.visibilityOf(
+      toastSuccessElement, 'Topic takes too long to get saved.');
```

Comment:
@kevintab95, Should we change this to "Topic saved success toast takes too long to appear!"? (Just to make the error log more precise? wdyt?)

Reply:
Yep, makes sense.